src/gui/gui_event.cpp

```diff
@@ -518,6 +518,26 @@ void event_desc_text::on_update(sys::state& state) noexcept {
 					l.color = delegate->black_text ? text::text_color::black : text::text_color::white;
 			}
 		}
+	} else if(std::holds_alternative<event::pending_human_p_event>(content)) {
+		auto phe = std::get<event::pending_human_p_event>(content);
+		auto imm = state.world.provincial_event_get_immediate_effect(phe.e);
+		if(imm) {
+			effect_description(state, contents, imm, trigger::to_generic(phe.p), trigger::to_generic(phe.p), phe.from_slot, phe.r_lo, phe.r_hi);
+			for(auto& l : delegate->internal_layout.contents) {
+				if(l.color == (delegate->black_text ? text::text_color::white : text::text_color::black)) //Invert colours
+					l.color = delegate->black_text ? text::text_color::black : text::text_color::white;
+			}
+		}
+	} else if(std::holds_alternative<event::pending_human_f_p_event>(content)) {
+		auto phe = std::get<event::pending_human_f_p_event>(content);
+		auto imm = state.world.free_provincial_event_get_immediate_effect(phe.e);
+		if(imm) {
+			effect_description(state, contents, imm, trigger::to_generic(phe.p), trigger::to_generic(phe.n), -1, phe.r_lo, phe.r_hi);
```

Comment:
why is the this slot different from the main slot here?

Reply:
because i am an idiot